Formula/recipes.rb

```diff
@@ -23,11 +25,16 @@ def install
                           "--disable-debug",
                           "--prefix=#{prefix}",
                           "--disable-autoar",
-                          "--disable-gspell"
+                          "--disable-gspell",
+                          "--disable-schemas-compile"
```

Comment:
Formulas that use glib schemas all end up being compiled into /usr/local/share/glib-2.0/schemas/gschemas.compiled, which is shared by all.

I we allow formulas to produce their own gschemas.compiled files you will get a `brew link` error, so it needs to happen during post-install.